extensions/ql-vscode/test/vscode-tests/minimal-workspace/databases.test.ts

```diff
@@ -574,6 +585,44 @@ describe("databases", () => {
     });
   });
 
+  describe("createSkeletonPacks", () => {
+    let mockDbItem: DatabaseItemImpl;
+
+    describe("when the language is set", () => {
+      it("should offer the user to set up a skeleton QL pack", async () => {
```

Comment:
Do we also need to check that we _don't_ do anything if the user doesn't have the `codeQL.codespacesTemplate` flag enabled?

Reply:
Chatted offline (wooohoo mini summit!). We're gonna check this in a subsequent PR for `openDatabase` tests: https://github.com/github/vscode-codeql/pull/2042